app/scripts/modules/core/src/widgets/cluster/clusterMatches.component.ts

```diff
@@ -23,5 +23,5 @@ class ClusterMatchesComponent implements IComponentOptions {
 }
 
 export const CLUSTER_MATCHES_COMPONENT = 'spinnaker.core.widget.cluster.clusterMatches.component';
-module(CLUSTER_MATCHES_COMPONENT, [])
+module(CLUSTER_MATCHES_COMPONENT, [ require('core/account/accountTag.component').ACCOUNT_TAG_COMPONENT ])
```

Comment:
Shouldn't need this `require` here - the tag is getting pulled in by the `account.module.js` file.

(trying to get away from including module dependencies if they're directives/components, except via `*.module` files)

If it's there for DOM testing, you can just import it into the spec file under test.